README.md

```diff
@@ -95,7 +90,7 @@ We've been working hard on improving and iterating on the IPFS Protocol to make
 - Complete Specification of the Protocol
 - Security Audits and Security Program
 - Go and JS IPFS enable modern IPFS data formats ([UnixFSv2](https://github.com/ipfs/unixfs-v2), [CIDv1](https://github.com/ipld/cid), raw blocks) by default
```

Comment:
If we are focusing on package managers, then reproducibility of data imports is crucial and should be mentioned on the roadmap.  
What I mean by reproduciblity of data imports: the same data should produce the same CID, even if IPFS project decides to change default hash, chunker settings, unixfs representation etc. Package managers use cases will require an easy way to define all "variables" upfront (eg. via cli flags) of embedd them in data itself (https://github.com/ipfs/unixfs-v2/issues/15), to ensure old CIDs remain reproducible. 
 
```suggestion
- Go and JS IPFS enable modern IPFS data formats ([UnixFSv2](https://github.com/ipfs/unixfs-v2), [CIDv1](https://github.com/ipld/cid), raw blocks) by default and in a reproducible way
```

Ref. 
- https://wiki.debian.org/ReproducibleBuilds